src/main/java/com/codejoust/main/game_object/Game.java

```diff
@@ -28,4 +28,7 @@
 
     // Boolean to hold whether all users have solved the problem.
     private Boolean allSolved = false;
+
+    // Boolean to hold whether the host prematurely ended the game
```

Comment:
nit, personal preference: "Prematurely" makes it sound like the host did it on accident, so I made a small suggested change to clarify the wording.
```suggestion
    // Boolean to hold whether the host ended the game early
```